src/dovi/rpu_info.rs

```diff
@@ -333,17 +382,248 @@ impl RpusListSummary {
             .map(|target_nits| format!("{target_nits} nits"))
             .collect();
 
+        let l5_blocks: Vec<_> = rpus
+            .iter()
+            .filter_map(|rpu| {
+                rpu.vdr_dm_data.as_ref()?.get_block(5).and_then(|block| {
+                    if let ExtMetadataBlock::Level5(l5) = block {
+                        Some(l5)
+                    } else {
+                        None
+                    }
+                })
+            })
+            .unique()
+            .collect();
+
+        type L5Mapping = (&'static str, fn(&ExtMetadataBlockLevel5) -> u16);
+        let l5_mappings: [L5Mapping; 4] = [
+            ("TOP", |l5| l5.active_area_top_offset),
+            ("BOTTOM", |l5| l5.active_area_bottom_offset),
+            ("LEFT", |l5| l5.active_area_left_offset),
+            ("RIGHT", |l5| l5.active_area_right_offset),
```

Comment:
I would rather see a format like `top=280, bottom=280, left=0, right=0` 
It's more readable.

Examples of the cases:
```
top=280, bottom=280, left=0, right=0 # default case
top=N/A, bottom=N/A, left=N/A, right=N/A # no L5
top=0..280, bottom=0..280, left=0, right=0 # min max range
```

Not sure about the range format but I like it.